pkgs/applications/graphics/gimp/default.nix

```
@@ -80,40 +101,57 @@ stdenv.mkDerivation (finalAttrs: {
 
     # Use absolute paths instead of relying on PATH
     # to make sure plug-ins are loaded by the correct interpreter.
-    ./hardcode-plugin-interpreters.patch
-
-    # GIMP queries libheif.pc for builtin encoder/decoder support to determine if AVIF/HEIC files are supported
-    # (see https://gitlab.gnome.org/GNOME/gimp/-/blob/a8b1173ca441283971ee48f4778e2ffd1cca7284/configure.ac?page=2#L1846-1852)
-    # These variables have been removed since libheif 1.18.0
-    # (see https://github.com/strukturag/libheif/commit/cf0d89c6e0809427427583290547a7757428cf5a)
-    # This has already been fixed for the upcoming GIMP 3, but the fix has not been backported to 2.x yet
-    # (see https://gitlab.gnome.org/GNOME/gimp/-/issues/9080)
-    ./force-enable-libheif.patch
+    # TODO: This now only appears to be used on Windows.
+    (replaceVars ./hardcode-plugin-interpreters.patch {
+      python_interpreter = python.interpreter;
+      PYTHON_EXE = null;
+    })
+
+    # D-Bus configuration is not available in the build sandbox
+    # so we need to pick up the one from the package.
+    (replaceVars ./tests-dbus-conf.patch {
+      session_conf = "${dbus.out}/share/dbus-1/session.conf";
+    })
   ];
 
   nativeBuildInputs =
     [
-      autoreconfHook # hardcode-plugin-interpreters.patch changes Makefile.am
+      meson
+      ninja
       pkg-config
-      intltool
       gettext
-      makeWrapper
-      gtk-doc
-      libxslt
+      wrapGAppsHook3
+      libxslt # for xsltproc
+      gobject-introspection
+      perl
+      vala
+
+      # for docs
+      gi-docgen
+
+      # for tests
+      desktop-file-utils
+      dbus
+    ]
+    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [
```

```suggestion
    ++ lib.optionals stdenv.hostPlatform.isLinux [
```
that one is linux only https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/xvfb-run/default.nix#L74